doc/copp/Copp_Manager_Redesign_test_plan.md

```diff
@@ -0,0 +1,134 @@
+# Copp Manager Redesign Test Plan
+
+## Related documents
+
+| **Document Name** | **Link** |
+|-------------------|----------|
+| CoPP Manager Redesign HLD | []|
+
+
+## Overview
+
+In the current design of Copp Manager, before installing a trap, CoppMgr checks if one of the following requirements is fulfilled:
```

Comment:
Can we modify 'current design' to 'previous design'?

Reply:
Fixed